usage/keyboard-shortcuts.md

```diff
@@ -45,7 +45,6 @@ The FreeTube client can also be controlled with shortcuts you might already be f
 
 | Key                                     | Function                               |
 | --------------------------------------- | -------------------------------------- |
-| Command/Ctrl + Q                        | Quit FreeTube                          |
 | Command/Ctrl + R                        | Reload FreeTube                        |
```

Comment:
what about `Command + Q (MacOS)/Alt + f4(Other OS)`?

Reply:
Alt + f4 works on Linux too?

Reply:
Yup, tried it on Ubuntu.

Reply:
`Command + Q (MacOS)/Alt + f4(Windows & Linux)`

There are so many OSes in the world, let's not confuse people

Reply:
Reason i did it like that is another shortcut is also listed in this way:

`Command + Y (MacOS)/Ctrl + H (Other OS) | Navigate to History`

I didn't wanted to have any inconsistencies in our docs because that could potentially lead to confusion. So i think we should change this one as well but let me know what u think.

We also clearly state the following in our readme:

[...] Available for **Windows, Mac & Linux** thanks to Electron.

Based on that i think that's another +1 why it should say (Windows & Linux), if applicable to both.